universal-login-sdk/test/fixtures/basicSDK.js

```diff
@@ -25,4 +42,6 @@ export const transferMessage = {
   value: utils.parseEther('0.5').toString()
 };
 
-
+export const addKeyMessage = {
+  ...MESSAGE_DEFAULTS
```

Comment:
That seems a bit misleading. Why addKeyMessage would have just MESSAGE_DEFAULTS. Perhaps it is just addKeyOptions or just messageDefaults?

Reply:
✅